nx-monorepo/README.md

```diff
@@ -0,0 +1,195 @@
+# Nx Monorepo
+
+This example shows how to use Nx to organize a mono repo and track dependencies.
+
+The example consists of the following components:
+
+```
+ - packages:
+     - s3folder: ComponentResource that manages a S3 bucket
+     - website-deploy: ComponentResource resource that manages files in a S3 bucket
+     - website-builder: Mock website generator that creates HTML output
+ - infra: Pulumi program that uses the s3folder and website ComponentResources to deploy a website
+```
+
+To deploy the latest version of the website, we need to respect the certain dependencies:
```

Comment:
I would word this as

To deploy the latest version of the website, we need to respect the following dependencies:

Reply:
Thanks!